protocols/imap/src/main/java/org/apache/james/imap/api/ImapConstants.java

```diff
@@ -152,6 +154,10 @@ public interface ImapConstants {
 
     String STATUS_SIZE = "SIZE";
 
+    String STATUS_DELETED = "DELETED";
+
+    String STATUS_DELETED_STORAGE = "DELETED_STORAGE";
```

Comment:
```suggestion
    String STATUS_DELETED_STORAGE = "DELETED-STORAGE";
```

`The DELETED and DELETED-STORAGE status data items allow for estimation of the amount of resources that could be freed by an EXPUNGE on a mailbox.`